src/App.vue

```diff
@@ -578,13 +579,18 @@ export default {
         throw new Error(`Invalid ${filingName} filing - filing status`)
       }
 
-      const nameRequest = filing[filingName].nameRequest
+      // NB: different object from actual NR
+      const nameRequest = filing[filingName].nameRequest as {
+        legalName?: string
+        legalType: CorpTypeCd
+        nrNumber: string
+      }
```

Comment:
This is the partial name request object attached to an IA or Registration filing.